bin/varnishtest/tests/v00018.vtc

```diff
@@ -9,7 +9,7 @@ varnish v1 -vcl {
 
 varnish v1 -errvcl {Variable is read only.} {
 	backend b { .host = "127.0.0.1"; }
-	sub vcl_miss { set now = 1000; }
+	sub vcl_miss { set now += 1s; }
 }
```

Comment:
This looks like something that should just have been committed ?

Reply:
done: cfed0152c0b846285b18c897e96f1e4e4b56a98d